app/workers/check_job.rb

```diff
@@ -1,5 +1,5 @@
-class CheckWorker
```

Comment:
The worker file need moving from `app/workers` to `app/sidekiq` (it's not mandatory to do this to upgrade to Sidekiq 7, but it's where Sidekiq will generate any future job files, so makes sense to keep them consistent).

You also need to add an alias (like in https://github.com/alphagov/asset-manager/pull/1504), otherwise previously queued jobs will fail.

Reply:
done :)